crud/readview.lua

```diff
@@ -0,0 +1,303 @@
+local fiber = require('fiber')
+local checks = require('checks')
+local errors = require('errors')
+local tarantool = require('tarantool')
+
+
+local stash = require('crud.common.stash')
+local utils = require('crud.common.utils')
+local sharding = require('crud.common.sharding')
+local select_executor = require('crud.select.executor')
+local select_filters = require('crud.compare.filters')
+local dev_checks = require('crud.common.dev_checks')
+local schema = require('crud.common.schema')
+local select = require('crud.select.compat.select')
```

Comment:
CI fails on 1.10.6 due to this line. Use the same approach as in `crud/select.lua`, but in case `has_merger` is `false` return some dummy module table like `{ new = function() return nil, ReadviewError:new("Tarantool does not support readview") end}`

Reply:
Maybe it's worth to add `is 2.11 and enterprise` check here and not on storage-level. It may be a bit unfair since router and storage may have different versions, but I think it's weird to try to use readview with half-baked cluster.

Reply:
Let's also add a test for all readview non-supported versions that "not supported" error is returned.

Reply:
added